console.lua

```diff
@@ -280,6 +280,94 @@ commands = {{
         G:update_shop()
         return "Reset shop."
     end
+}, {
+    name = "value",
+    source = "debugplus",
+    shortDesc = "Get and modify highlighted card values",
+    desc = "Lets you get or modify the values of the currently hovered card",
+    exec = function (args, rawArgs, dp)
+        local unmodified_vals = {
+            bonus = 0,
+            perma_bonus = 0,
+            extra_value = 0,
+            p_dollars = 0,
+            h_mult = 0,
+            h_x_mult = 0,
+            h_dollars = 0,
+            h_size = 0,
+            d_size = 0,
+            hands_played_at_create = 0,
+            mult = 0,
+            x_mult = 1,
+            e_mult = 0,
+            ee_mult = 0,
+            eee_mult = 0,
+            x_chips = 0,
+            e_chips = 0,
+            ee_chips = 0,
+            eee_chips = 0,
+            t_mult = 0,
+            t_chips = 0,
+        }
+        local ignore_vals = {
+            name = true,
+            set = true,
+            order = true,
+            consumeable = true
+        }
+        local hovered = G.CONTROLLER.hovering.target
+        if hovered:is(Card) then
+            if args[1] == "get" then
+                for k, v in pairs(hovered.ability) do
+                    if (not ignore_vals[k]) and (not unmodified_vals[k] or unmodified_vals[k] ~= hovered.ability[k]) then
+                        if k == "hyper_chips" or k == "hyper_mult" then
+                            if hovered.ability[k][1] ~= 0 or hovered.ability[k][2] ~= 0 then
+                                print(k, hovered.ability[k][1], hovered.ability[k][2])
+                            end
+                        elseif type(hovered.ability[k]) == "table" then
+                            for kk, vv in pairs(hovered.ability[k]) do
+                                print(k, kk, vv)
+                            end
+                        elseif hovered.ability[k] ~= "" then
+                            print(k, hovered.ability[k])
```

Comment:
For commands, they should always return something, and if they absolutly must print before/after the return they should use dp.handleLog. theres some logistics about the differences between how print and handlelog work, but the basics are no log level control and it won't be marked as a command so it can be hidden with some of the config options.